cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -413,13 +404,14 @@ def upload_rna_fraser_outrider_to_scout(
                 customer_case = status_db.get_case_by_internal_id(dna_case_id)
                 if dry_run:
                     continue
-                self.scout_api.upload_rna_fraser_outrider(
-                    fraser_file_path=rna_fraser.full_path,
-                    outrider_file_path=rna_outrider.full_path,
-                    case_id=dna_case_id,
-                    customer_case_name=customer_case.name,
-                    cust_id=customer_case.customer.internal_id,
-                )
+                if rna_fraser or rna_outrider:
+                    self.scout_api.upload_rna_fraser_outrider(
+                        fraser_file_path=rna_fraser.full_path,
+                        outrider_file_path=rna_outrider.full_path,
+                        case_id=dna_case_id,
+                        customer_case_name=customer_case.name,
+                        cust_id=customer_case.customer.internal_id,
+                    )
```

Comment:
I found the logging a bit confusing, maybe this can help? feel free to disregard
```suggestion
                    )
                else:
                    LOG.warning("No fraser nor outrider files found to upload")
```

Reply:
I don't think this improves the loging situation. Will refrain from adding